src/scss/components/person/_person-teasers.scss

```diff
@@ -0,0 +1,47 @@
+@charset "UTF-8";
+
+.ptype-stanford-entity,
+.paragraph--type--stanford-entity {
+
+  .ds-entity--stanford-person {
```

Comment:
@jenbreese should the Person Teaser span full width for single column/responsive? 🤔 

For the screenshot, my screen width was 640px
<img width="741" alt="image" src="https://github.com/SU-SWS/taiwan_hub_subtheme/assets/34019925/90b0bfb4-60ac-49d5-86f0-367429df2a48">

Reply:
@rebeccahongsf No, Kerri wanted the person teaser only to have the max width of 400. I forgot to put in the jira ticket to the PR. I'm sorry about that. https://stanfordits.atlassian.net/browse/THUB-21